### ISSUE TITLE
 Expecting this URL to be successfully parsed

### ISSUE THREAD

**Issue by zsoltforray** [python-validators/validators#274]
Hi, 

I have found that the URL validation of _http://**verveacu**--fiserv.sandbox.lightning.force.com/auraCmpDef_ returns False. I understand that the URL format is validated so I have tested other URL having the same format like:
_http://**xn**--fiserv.sandbox.lightning.force.com/auraCmpDef_ and the URL validator gives back True. So I have just made a little change again and tested this _https://**xm**--fiserv.sandbox.lightning.force.com/auraCmpDef_, which returns False again. 

I have been using the validators for quite a long time, but I cannot find explanation for this behavior. I think that the validation of _http://**verveacu**--fiserv.sandbox.lightning.force.com/auraCmpDef_ should return True.

BR: Zsolt

**Comment by joe733** [comment]
Fixed in #264.

```python
Python 3.8.0 (default, Apr 24 2023, 15:11:36) 
[GCC 12.2.1 20230201] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import validators
>>> validators.__version__
'0.21.1'
>>> validators.url("http://verveacu--fiserv.sandbox.lightning.force.com/auraCmpDef")
True
```

ref: https://github.com/python-validators/validators/issues/267 & https://github.com/python-validators/validators/issues/268

**Comment by zsoltforray** [comment]
Thank you very much @joe733
